### PR TITLE
(#1106) NoNullsTest.java: `Rule` replaced with `Throws`

### PR DESCRIPTION
**Pull request by iakunin**
NoNullsTest.java: `Rule` replaced with `Throws` for https://github.com/yegor256/cactoos/issues/1106

### CODE REVIEW

File: src/test/java/org/cactoos/iterator/NoNullsTest.java [hunk 1]
```diff
@@ -75,20 +67,20 @@ public String toString() {
...
+        new Assertion<>(
+            "nth throws error if null",
+            () -> {
+                final Iterator<String> itr = new NoNulls<>(
+                    new IteratorOf<>("a", "b", null, "c")
+                );
+                while (itr.hasNext()) {
+                    itr.next();
+                }
+                return null;
```
Comment (iakunin) [inline src/test/java/org/cactoos/iterator/NoNullsTest.java]: I'm not sure about this `return null;`, but I have no better solution

**Comment by 0crat** [comment]
Job [#1187](https://github.com/yegor256/cactoos/issues/1187) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1187](https://github.com/yegor256/cactoos/issues/1187) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/25d8b1d3-5597-4427-8373-e95e0acb8666); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/iterator/NoNullsTest.java (hunk 1 above)
Comment (fabriciofx) [reply src/test/java/org/cactoos/iterator/NoNullsTest.java]: @iakunin you can't use anything else? Like an `Object`? If I understood correctly, this `return null` is never reached. Am I right?

Comment (iakunin) [reply src/test/java/org/cactoos/iterator/NoNullsTest.java]: @fabriciofx of course I can use an `Object`. Yes, `return null` is never reached.

Comment (iakunin) [reply src/test/java/org/cactoos/iterator/NoNullsTest.java]: @fabriciofx of course I can use an Object. Yes, return null is never reached.

**Comment by iakunin** [comment]
@fabriciofx could you review it once again, please?

**Comment by iakunin** [comment]
@fabriciofx

**Comment by iakunin** [comment]
@fabriciofx could you review it once again, please?

**Comment by paulodamaso** [comment]
@iakunin please solve conflicts

**Comment by iakunin** [comment]
@paulodamaso I merged `upstream/master` and realized that this issue already been solved in https://github.com/yegor256/cactoos/pull/1171 by @vzurauskas

To be honest, I don't completely understand why that's happened. 

@paulodamaso what should I do in this case? Just close this PR?

**Comment by paulodamaso** [comment]
@iakunin Yes, please, close this then

**Comment by 0crat** [comment]
Code review was too long (17 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1187](https://github.com/yegor256/cactoos/issues/1187) is now out of scope

**Comment by 0crat** [comment]
Pull request [#1187](https://github.com/yegor256/cactoos/issues/1187) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by WhimsySoftware** [comment]
@fabriciofx please make sure you will try find at least 3 possible issues during next CR, thanks

**Comment by sereshqua** [comment]
@fabriciofx please make sure you will try find at least 3 possible issues during next CR, thanks

**Comment by fabriciofx** [comment]
@sereshqua ok. Thanks!

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by sereshqua** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1187#issuecomment-538657894))

@sereshqua This is what I know about this job in [C63314D6Z](https://www.0crat.com/p/C63314D6Z), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1187](https://github.com/yegor256/cactoos/issues/1187) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231187%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
> @0crat quality acceptable  ([here](https://github.com/yegor256/cactoos/issues/1187#issuecomment-538658182))

@sereshqua There is no quality review for [#1187](https://github.com/yegor256/cactoos/issues/1187), no performer

**Pull request closed without merging**